### PR TITLE
feat(get-platform): add FreeBSD 15 as a new platform

### PR DESCRIPTION
**Pull request by obiwac**

### CODE REVIEW

File: packages/get-platform/src/binaryTargets.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export type BinaryTarget =
   | 'freebsd12'
   | 'freebsd13'
   | 'freebsd14'
+  | 'freebsd15'
```
Comment (aqrln) [inline packages/get-platform/src/binaryTargets.ts:29]: You are also missing it in the `binaryTargets` array below

**Review by aqrln — COMMENTED** [review]
Thank you! Looks good except one comment

File: packages/get-platform/src/binaryTargets.ts (hunk 1 above)
Comment (obiwac) [reply packages/get-platform/src/binaryTargets.ts:29]: yes, my bad, it was unstaged ;)

**Review by aqrln — APPROVED** [review]
LGTM, thank you!

**Pull request merged**